apps/frontend/src/store/snackbar.ts

```diff
@@ -23,22 +23,42 @@ export class Snackbar extends VuexModule {
   message = '';
   error = false;
   show = false;
+
   @Action
   notify(message: string) {
     this.context.commit('SET_ERROR', false);
     this.context.commit('SET_MESSAGE', message);
     this.context.commit('SET_VISIBILITY', true);
   }
+
   @Action
   failure(message: string) {
     this.context.commit('SET_ERROR', true);
     this.context.commit('SET_MESSAGE', message);
     this.context.commit('SET_VISIBILITY', true);
   }
+
+  @Action
+  HTTPFailure(error: any) {
+    console.log(error.response.data.message);
+    if (typeof error.response.data.message === 'object') {
+      this.notify(
+        error.response.data.message
+          .map(function capitalize(c: string) {
+            return c.charAt(0).toUpperCase() + c.slice(1);
+          })
+          .join(', ')
+      );
+    } else {
+      this.notify(error.response.data.message);
```

Comment:
Should this be using `failure` instead of `notify`? This seems like an error not just a notification.